docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: "3.8"
+
+services:
+  todo-db:
+    image: mariadb:latest
+    container_name: todo-db
+    environment:
+      MARIADB_ROOT_PASSWORD: S3cret!
+      MARIADB_DATABASE: todo
+      MARIADB_USER: todo
+      MARIADB_PASSWORD: todo
+    volumes:
+      - todo-db:/var/lib/mysql
+
```

Comment:
Add `port` configuration section to bind host port 3307 to maridb container port 3306

Reply:
done sir.
![image](https://github.com/LombokAcademy/todo-app/assets/95129603/a7d819db-bfd5-4e14-b4db-d601c04d8f6e)